lib/rails/generators/alchemy/install/files/application.html.erb

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%= @page.language_code %>">
```

Comment:
My thought was that we only need the language code, not the country. I just looked it up at w3c:

> The golden rule when creating language tags is to keep the tag as short as possible. Avoid region, script or other subtags except where they add useful distinguishing information. For instance, use ja for Japanese and not ja-JP, unless there is a particular reason that you need to say that this is Japanese as spoken in Japan, rather than elsewhere.

https://www.w3.org/International/articles/language-tags/

So, we could use the country code, but they strongly recommend to keep it short.